README.md

```diff
@@ -139,20 +113,62 @@ Stack:
 
 Read more about the CLI in the [documentation](https://docs.dyrector.io/get-started/cli).
 
+## Non-development
+
+You can set up dyrectorio for self-hosting purposes with the [docker-compose](https://github.com/dyrector-io/dyrectorio/blob/develop/docker-compose.yaml) file located in the root folder.
+
 ## Hosted version (SaaS)
 
-We are planning to support a hosted version in the near future.
+Besides the self-hosted instance of dyrectorio, you can check out the platform's alpha at [app.dyrectorio.com](https://app.dyrector.io). The platform is still in progress, we might refresh database time-to-time, so it's not recommended to use for production. In case you're interested in using dyrectorio in production, reach out to us via email and we'll set up a stable instance for you.
```

Comment:
`might reset the database, so it's not recommended for production yet. In case you are interested about using dyrectorio in production...`

Reply:
Resolved.